src/components/PostInfo/PostInfo.tsx

```diff
@@ -1,5 +1,44 @@
 import React from 'react';
+import { CommentList } from '../CommentList';
+import { Post } from '../types/Post';
```

Comment:
Moving types folder to the src folder would make more sense , because it isn't a component.